app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -8,7 +8,9 @@ def create_in_provider(manager_id, params)
       tower_object = provider_collection(manager).create!(params)
 
       refresh(manager)
-      find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
+      target = find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
+      refresh(target)
```

Comment:
Yeah if this is what we have to do to fix a blocker I'm :+1: with it

Longer term I agree refresh_new_target and native op workflow can help here; I think we should also look at the inconsistent behavior between manager/full refresh and targeted refresh.